pipeline_tools/get_analysis_metadata.py

```diff
@@ -1,11 +1,21 @@
 import argparse
 import json
-from requests.auth import HTTPBasicAuth
 from cromwell_tools import cromwell_tools
+from requests.auth import HTTPBasicAuth
+
 from pipeline_tools.http_requests import HttpRequests
 
 
-def get_workflow_id(analysis_output_path):
+def _get_caas_backend():
+    """Get the url to the Cromwell-as-a-service backend.
+
+    This is a temporary function abstracted from the other functions which are hard-coding the url. FIXME: it doesn't
+    make sense to hard-code the CaaS url anymore once we migrate to use caas-prod as backend.
+    """
+    return 'https://cromwell.caas-dev.broadinstitute.org/api/workflows/v1'
```

Comment:
Since this returns a constant value, how about setting a variable instead, e.g: 

`CAAS_URL = 'https://cromwell.caas-dev.broadinstitute.org/api/workflows/v1'`

I don't like the fact that this is hard-coded either, since we will have to change this to the caas-prod URL in addition to changing the Lira config. But the alternative involves a lot of code changes to parameterize this, and I don't expect this to change often so I think it's okay to keep it defined here.

Reply:
I think we should pass the cromwell url in as a parameter where it's needed, and not hard code it in pipeline-tools, even temporarily.

Reply:
This was copied over, but I will make it as a parameter in static inputs as suggested.